data/json/npcs/island_prison/prisoners.json

```diff
@@ -0,0 +1,513 @@
+    "responses": [
+      {
+        "text": "No, of course not!  Sorry, I'm leaving.",
+        "topic": "TALK_DONE",
+        "condition": { "not": { "u_has_var": "talked_to_greeter", "type": "dialogue", "context": "annoyed_greeter", "value": "yes" } }
+      },
+      { "text": "Hey, <fuck_you>!", "topic": "TALK_DONE", "effect": "hostile" }
+    ]
+  },
+  {
+    "id": "TALK_PRISONER_ANGRY",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_has_var": "talked_to_greeter",
+      "type": "dialogue",
+      "context": "annoyed_greeter",
+      "value": "yes",
+      "no": "<get_lost>",
+      "yes": "Are you deaf or stupid?  I said I don't give a fuck to you and your problems.  Bother me one more time, and you'll regret that.  <get_lost>"
+    },
+    "speaker_effect": { "effect": { "u_add_var": "talked_to_greeter", "type": "dialogue", "context": "angry_greeter", "value": "yes" } },
+    "responses": [
+      {
+        "text": "Ok, sorry, I promise I won't bother you anymore.",
+        "topic": "TALK_DONE",
+        "condition": { "not": { "u_has_var": "talked_to_greeter", "type": "dialogue", "context": "angry_greeter", "value": "yes" } }
+      },
+      { "text": "Hey, <fuck_you>!", "topic": "TALK_DONE", "effect": "hostile" }
+    ]
+  },
+  {
+    "id": "TALK_PRISONER_READY_TO_MUG",
+    "type": "talk_topic",
+    "dynamic_line": "You think you're immortal, huh, <name_b>?",
+    "responses": [
+      { "text": "Gee, what's the matter with you?  Okay, okay, I'm leaving.", "topic": "TALK_PRISONER_MUG" },
+      {
+        "text": "It would be you who will regret talking to me like that.",
+        "topic": "TALK_DONE",
+        "effect": "hostile"
+      }
+    ]
+  },
+  {
+    "id": "TALK_PRISONER_MUG",
+    "type": "talk_topic",
+    "dynamic_line": [ "Not so fast, I think you need to pay for breaking your promise and bothering me.  Give me your shit, now!" ],
+    "responses": [
+      { "text": "Please don't hurt me!  Take all that you want.", "topic": "TALK_MUG" },
+      { "text": "What, <no>!", "topic": "TALK_DONE", "effect": "hostile" }
+    ]
+  },
+  {
+    "id": "TALK_PRISONER_LEADER_GREET",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_has_var": "prisoner_leader_mission",
+      "type": "mission",
+      "context": "accepted",
+      "value": "yes",
+      "yes": "So, what are you waiting for?",
+      "no": {
+        "u_has_var": "prisoner_leader_mission",
+        "type": "mission",
+        "context": "completed",
+        "value": "yes",
+        "yes": "I don't know if you know, but there are sewers underneath the prison.  I was planning the escape long before <the_cataclysm>, and while I was working on cleaning the sewers, I noticed there is a damaged section of the wall.  There was a flow of fresh air coming out of this section, so I think it's leading to the surface.  It could be you way to the freedom.  Feel free to use it.",
```

Comment:
```suggestion
        "yes": "I don't know if you know, but there are sewers underneath the prison.  I was planning an escape long before <the_cataclysm>, and while I was working on cleaning the sewers, I noticed a damaged wall section. 
 There was a flow of fresh air coming out of it, so I think it's leading to the surface.  It could be your way to freedom.  Feel free to use it.",
```